eternalcore-core/src/main/java/com/eternalcode/core/feature/home/messages/PLHomeMessages.java

```diff
@@ -40,4 +40,27 @@ public class PLHomeMessages extends OkaeriConfig implements HomeMessages {
         Notice.chat("<red>► <white>Usunięto dom <red>{HOME} <white>dla gracza <red>{PLAYER}<white>!");
     public Notice homeListAsAdmin =
         Notice.chat("<green>► <white>Lista domów gracza <green>{PLAYER}<white>: <green>{HOMES}!");
+    public Notice noHomesOnListAsAdmin =
+        Notice.chat("<red>► <dark_red>Gracz <red>{PLAYER} <dark_red>nie posiada domów.");
+
+    @Comment({
+        " ",
+        "# {PLAYER} - nick właściciela domu na który przeteleportowano się za pomocą /homeadmin home",
+        "# {HOME} - nazwa domu"
+    })
+    public Notice teleportedAsAdmin =
+        Notice.chat("<green>► <white>Przeteleportowano do domu gracza: <green>{PLAYER}<white> - <green>{HOME}!");
+
+    @Comment({
+        " ",
+        "# Wiadomość wysyłana do gracza który nie wpisał poprawnie agumentów do komend /homeadmin"
+    })
+    public Notice missingArgument = Notice.chat("<dark_red>✘ <red>Brakujący argument! Proszę podać: <nazwa gracza> i <nazwa domu>");
```

Comment:
nie wiem czy to dobry pomysł bo jedyne co zyskujemy na razie to +1 field w konfigu i mniej informacji zwrotnej dla admina. Pamiętajmy że jednak to jest też narzędzie dla admina